lib/src/kbasesearchengine/common/JsonTokenUtil.java

```diff
@@ -161,19 +162,15 @@ public static Object getCurrentTokenPrimitive(JsonParser jts,
         } else if (t == JsonToken.VALUE_TRUE) {
             return true;
         } else {
-            throw new IOException("Unexpected token type: " + t);
+            throw new ObjectParseException("Unexpected token type: " + t);
         }
     }
 
-    public static String prettyPrint(Object obj) {
-        try {
+    public static String prettyPrint(Object obj) throws IOException {
```

Comment:
Is this code even used anywhere? Delete maybe?

Reply:
Removed.